src/coreComponents/mesh/generators/InternalWellGenerator.hpp

```diff
@@ -30,7 +30,7 @@ namespace geosx
  *
  * This class processes the data of a single well from the XML and generates the well geometry
  */
-class InternalWellGenerator : public MeshGeneratorBase
+class InternalWellGenerator : public dataRepository::Group
```

Comment:
Try to hide `InternalWellGenerator` in the other parts of the code: we should be using `WellBlockABC` instead. That way we can separate the data from the producer.

Reply:
I think it's done.